bundles/com.zeligsoft.base.zdl/src/com/zeligsoft/base/zdl/util/ZDLCopier.java

```diff
@@ -655,4 +658,111 @@ public EObject remove(Object key) {
 	public Set<Entry<EObject, EObject>> entrySet() {
 		return map.entrySet();
 	}
+	
+	class ZDLEcoreCopier extends EcoreUtil.Copier{
```

Comment:
Question: ZDLEcoreCopier is a nested class. Without `static`, instances have access to members of the containing class. Looking at the code, I don't see any such usage. Non-static nested classes have weird behaviour. If adding a `static` qualifier doesn't break anything, we should do it.

Reply:
Will make this change.